src/drv_oculus_rift/rift.c

```diff
@@ -440,21 +440,24 @@ static void get_device_list(ohmd_driver* driver, ohmd_device_list* list)
 			continue;
 
 		while (cur_dev) {
-			if(rd[i].iface == -1 || cur_dev->interface_number == rd[i].iface){
-				ohmd_device_desc* desc = &list->devices[list->num_devices++];
+			// This is needed because other companies (eg: VR-Tek) also use the Oculus DK1 USB ID
+			if (wcscmp(cur_dev->manufacturer_string, L"Oculus VR, Inc.")==0) {
```

Comment:
Ugh, this is ugly. I have no idea about DK1, but at least for DK2 and CV1 this manufacturer string seems to be correct. Please fold this into the existing if statement to avoid all the indentation changes.

Reply:
Yeah i want to double check all the oculus devices we currently support to make sure the ID is correct

Reply:
Good idea. I just pushed a commit that folds the check into the existing if statement. That's much less churn now.
Hopefully that manufacturer string is consistent across all the Oculus headsets, because it's at least a simple uglification ;)

Reply:
Thanks, squashed together, the change looks good now.